### PR TITLE
Odin

### PR DESCRIPTION
**Pull request by WalterAlvarez92**
Should be good?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -89,6 +89,7 @@ With this exercise we are attempting to collect the GitHub accounts of all stude
 
   #### Spring 2018 Class
   * Joe Jeppi: [github.com/jjeppi](https://github.com/jjeppi)
+  * Walter Alvarez: [github.com/WalterAlvarez92](https://github.com/WalterAlvarez92)
```
Comment (tkraak) [inline README.md:92]: @WalterAlvarez92 ... bring the line starting with `https` up so that everything is on *one* line

Comment (WalterAlvarez92) [reply README.md:92]: I brought the line up, but dont know if i have the most u to date

Comment (WalterAlvarez92) [reply README.md:92]: *up

Comment (tkraak) [reply README.md]: Where is the commit for bringing the line up?

Comment (WalterAlvarez92) [reply README.md:92]: position change

**Pull request closed without merging**